openc3/python/openc3/models/model.py

```diff
@@ -33,43 +35,40 @@ def store_queued(cls):
     # without primary_key to support other class methods.
 
     @classmethod
-    def get(cls, primary_key, name):
+    def get(cls, primary_key: str, name: str):
         """@return [Hash|nil] Hash of this model or nil if name not found under primary_key"""
         json_data = cls.store().hget(primary_key, name)
-        if json_data:
-            return json.loads(json_data)
-        else:
-            return None
+        return json.loads(json_data) if json_data else None
 
     @classmethod
-    def names(cls, primary_key):
+    def names(cls, primary_key: str):
         """@return [Array<String>] All the names stored under the primary key"""
         keys = cls.store().hkeys(primary_key)
         keys.sort()
         return [key.decode() for key in keys]
 
     @classmethod
-    def all(cls, primary_key):
+    def all(cls, primary_key: str):
         """@return [Array<Hash>] All the models (as Hash objects) stored under the primary key"""
         base = cls.store().hgetall(primary_key)
         # decode the binary string keys to strings
-        hash = {k.decode(): v for (k, v) in base.items()}
-        for key, value in hash.items():
-            hash[key] = json.loads(value)
-        return hash
+        hash_ = {k.decode(): v for (k, v) in base.items()}
+        for key, value in hash_.items():
+            hash_[key] = json.loads(value)
+        return hash_
```

Comment:
Why the underscore: `hash_`

Reply:
hash is a built-in function in python, https://docs.python.org/3/library/functions.html#hash